src/js/providers/data-normalizer.ts

```diff
@@ -0,0 +1,15 @@
+export function qualityLevel(level: any): QualityLevel {
```

Comment:
Assumed this was `any` because it looks like it can be a PlaylistItemSource or a flash/shaka/hls manifest level.

Reply:
I might consider doing a union type between generic object and Quality level for this, that way we get type safety on the attrs that are there but allow others in as well.

example: https://github.com/jwplayer/jwplayer/pull/3602/files#diff-c1229db6acf267fd235b1924faf60e62R3